Resources/Private/Fusion/Controller/Backend/FormData/Preview.fusion

```diff
@@ -0,0 +1,42 @@
+prototype(PunktDe.Form.Persistence:Backend.FormData.Preview) < prototype(Neos.Fusion:Component) {
+
+    backToFormDataLink = Neos.Fusion:UriBuilder {
+        package = 'PunktDe.Form.Persistence'
+        controller = 'Backend\\FormData'
+        action = 'index'
+    }
+
+    headerFields = Neos.Fusion:Loop {
+        items = ${headerFields}
+        itemRenderer = ${'<th>' + item + '</th>'}
+    }
+
+    tableRows = Neos.Fusion:Loop {
+        items = ${formDataValues}
+        itemName = 'row'
+        itemRenderer = Neos.Fusion:Loop {
+            items = ${row}
+            itemRenderer =  ${'<td>' + item + '</td>'}
+            @process.wrapRow = ${'<tr>' + value + '</tr>'}
+        }
+    }
+
+    renderer = afx`
+        <legend>{formIdentifier}</legend>
+        <div class="neos-content neos-container-fluid">
+            <div class="neos-module-container">
+                <table class="neos-table">
+                    <thead>
+                    <tr>{props.headerFields}</tr>
+                    </thead>
+                    <tbody>
+                        {props.tableRows}
```

Comment:
Hier wäre ein Hinweis gut, falls es keine Daten gibt, dass erst Daten angezeigt werden, wenn ein Formular mit der entsprechenden Einstellung abgeschickt wird.

Reply:
Ein Hinweis ist hier nicht nötig, da die Preview je Formular aufgerufen wird. Einformulare wird nur angezeigt, wenn es dafür gespeicherte Daten gibt.  Man könnte aber in der initialen Ansicht vielleicht einen entsprechenden Hinweise hinzufügen.